insights/specs/datasources/get_satellite_missed_queues.py

```diff
@@ -0,0 +1,88 @@
+import re
+
+from insights.core.context import HostContext
+from insights.core.dr import SkipComponent
+from insights.core.plugins import datasource
+from insights.core.spec_factory import DatasourceProvider
+from insights.parsers.messages import Messages
+from insights.combiners.satellite_version import SatelliteVersion
+Messages.keep_scan('qpidd_node_not_found_errors', 'error Error on attach: Node not found')
```

Comment:
@bfahr, sure, I'll update the PR according to the suggestions, thanks for the review.

Reply:
@bfahr since I'll check all the lines in the message, do we need to concern that the time will be longer if there are too many lines in messages even we add filter?